improver_tests/nbhood/circular_kernel/test_GeneratePercentilesFromACircularNeighbourhood.py

```diff
@@ -146,10 +156,12 @@ def test_2d_slice(self):
                     [1.0, 1.0, 1.0, 1.0, 1.0],
                     [1.0, 1.0, 1.0, 1.0, 1.0],
                 ],
-            ]
+            ],
         )
         kernel = np.array([[0.0, 1.0, 0.0], [1.0, 1.0, 1.0], [0.0, 1.0, 0.0]])
-        cube = self.cube[0, 0, :, :]
+        cube = self.cube
+        cube.data[2, 2] = 0
+        cube = cube[:, :]
```

Comment:
Not sure what these three statements are doing.  What would happen if you just had `self.cube.data[2, 2] = 0`, and then called the plugin on `self.cube`?

Reply:
Done. Yes, for some reason I didn't think self.cube.data was liked so I did it in stages but it is completely fine so I've changed it all now.